tests/scripts/models/run_test.sh

```diff
@@ -217,7 +217,7 @@ download_tests()
 
         TEST_CACHE_PATH=$CACHE_ROOT_PATH/$TEST_NAME
         MODELFILE=$TEST_CACHE_PATH/$MODELFILE_NAME
-        MODELFILE_URL="$MODELFILE_SERVER/$MODELFILE_NAME"
+        MODELFILE_URL="$MODELFILE_SERVER_PATH/$MODELFILE_NAME"
```

Comment:
`MODELFILE_SERVER` is set by user and it is commonly used for model server setting. `MODELFILE_SERVER_PATH` is comes from `config.sh` and it can have different path for each model file.